src/components/SharedComponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -230,13 +266,33 @@ const VideoPlayer: FC<VideoPlayerProps> = (props) => {
         onPause={onPause}
         onError={onError}
         onTimeUpdate={onTimeUpdate}
+        onCanPlay={() => {
+          if (reloadingDueToErrors && videoIsPlaying) {
+            mediaElRef.current?.play();
+          }
+        }}
```

Comment:
This decides whether to resume playing the video after an error has occurred dependant on if it was playing previously.